lib/bullet.rb

```diff
@@ -71,7 +71,7 @@ def enable?
     end
 
     def app_root
-      @app_root ||= (defined?(::Rails.root) ? Rails.root.to_s : Dir.pwd).to_s
+      @app_root ||= (defined?(::Rails.root) && !::Rails.root.nil? ? Rails.root.to_s : Dir.pwd).to_s
```

Comment:
`Rails.root` supposed not to be `nil`

Reply:
oh, I get what you mean after reading issue #568, please add a comment here why `Rails.root` could be `nil` here, thanks.

Reply:
Just added, thanks for the feedback!